build-system/pr-check/validator-tests.js

```diff
@@ -39,6 +39,15 @@ const FILELOGPREFIX = colors.bold(colors.yellow(`${FILENAME}:`));
 const timedExecOrDie = (cmd) => timedExecOrDieBase(cmd, FILENAME);
 const timedExec = (cmd) => timedExecBase(cmd, FILENAME);
 
+function skipJavaValidator() {
+  // #28497: Java Validator tests are broken due to Ubuntu keyserver outage.
+  console.log(
+    `${FILELOGPREFIX} Skipping`,
+    colors.cyan('Java Validator Tests'),
+    'temporarily because the Ubuntu keyserver is down.'
+  );
+}
+
```

Comment:
No need for the skipped message IMO. Simply commenting out the lines should do.

Reply:
Done.